### PR TITLE
stub in files from domain inventory sheet

### PR DESCRIPTION
**Pull request by afeld**
Used the following script to create them, passing the CSV exported from [our domain list](https://docs.google.com/spreadsheets/d/12pfcEIEXaJTjIKex-3wnI89erIvgKf9B_XpGkDl6qsM/edit#gid=824448842) as an argument:

```python
import csv
import os.path
import sys

source = sys.argv[1]
with open(source) as csvfile:
    reader = csv.DictReader(csvfile)
    for row in reader:
        domain = row["Domain Name"].lower()
        if not domain:
            continue

        if row["Decommissioned"] == "TRUE":
            continue

        filename = f"{domain}.tf"
        if os.path.isfile(filename):
            continue

        print("Stubbing files…")
        print(domain)
        with open(filename, "w") as new_file:
            contents = """# managed by Corporate IT - request changes through
# https://gsa.servicenowservices.com/sp/?id=kb_article&sys_id=075cd185db26578058c2fd721f96193b
"""
            new_file.write(contents)
```

Did some manual checks and adjustment to make them accurate.

### CODE REVIEW

File: terraform/citizenscience.gov.tf [hunk 1]
```diff
@@ -0,0 +1 @@
+challenge.gov.tf
```
Comment (afeld) [inline terraform/citizenscience.gov.tf:1]: This is a symlink.

**Review by JJediny — APPROVED** [review]
LGTM

**Review by JJediny — CHANGES_REQUESTED** [review]
@afeld just missed that the script and the reason for its use here should probably be added as content in the repo vs just in the PR comments (python script and when it may be needed in the future)

**Comment by afeld** [comment]
I don't expect us to need the script again, hence not including it.

**Review by JJediny — APPROVED** [review]

**Pull request merged**